lib/uwb/include/uwb/UwbMacAddress.hxx

```diff
@@ -361,6 +361,15 @@ public:
      */
     UwbMacAddress(const UwbMacAddress& other);
 
+    /**
+     * @brief Construct a new UwbMacAddress object given an input string of
+     * semicolon-delimited bytes.
```

Comment:
It'd be nice if the delimiter was configurable and/or optional. For example, if someone wanted to specify `0A0B` on the command line, I don't see why we shouldn't allow it.

No need to hold up the review for this, but perhaps adding a TODO somewhere here indicating this would be good.